### ISSUE TITLE
add static category

### ISSUE THREAD

**Issue by sanderson** [nanobox-io/nanobox-guides#28]

**Comment by mohessaid** [comment]
Yes, I think this will help too. I use Nanobox to keep my operating system as clean as possible while using different languages and technologies. I work on static website generators and I would love to see them added here and contribute to them too.

**Comment by tylerflint** [comment]
That's a great suggestion @mohessaid 👍
